pkg/networkmanager/wireguard.jsx

```diff
@@ -117,7 +117,7 @@ export function WireGuardDialog({ settings, connection, dev }) {
                 const key = await cockpit.spawn(["wg", "pubkey"], { err: 'message' }).input(privateKey.trim());
                 setPublicKey(key.trim());
             } catch (e) {
-                console.error(e.message);
+                console.error("Failed to call wg pubkey:", e.message);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-101817-f8bf272a-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)